app/helpers/secure_headers_helper.rb

```diff
@@ -0,0 +1,34 @@
+module SecureHeadersHelper
+  def backwards_compatible_javascript_tag(*args, **opts, &block)
+    if FeatureManagement.rails_csp_tooling_enabled?
+      javascript_tag(*args, opts.merge(nonce: true), &block)
+    else
+      nonced_javascript_tag(*args, **opts, &block)
+    end
```

Comment:
should we include `IdentityConfig.store.disable_csp_unsafe_inline` in here now that we added it in the other PR? https://github.com/18F/identity-idp/pull/5844/files#diff-66cc7258886071348febbf60314501e02948b7d31e616fe88121ccc338ddb449R38

```suggestion
    if FeatureManagement.rails_csp_tooling_enabled? || IdentityConfig.store.disable_csp_unsafe_inline
      javascript_tag(*args, opts.merge(nonce: true), &block)
    else
      nonced_javascript_tag(*args, **opts, &block)
    end
```

Reply:
yeah, i can work it in. i think it may make more sense in the header, right?

Reply:
added in 2332f79c7ce2fad1247d02c65ce6b740cbcccd47